Update static embedding iframe resizer (#49609)
metabase/metabase: docs/embedding/static-embedding.md

></iframe>
```

> Note: we recommend that you use up to version 4.3.2 of iframe-resizer on your application due to licensing changes after that

Reviewer: ```suggestion
Due to iframe-resizer's licensing changes, we recommend that you use iframe-resizer version 4.3.2 or lower. 
```
